packages/api/orm/entity/TaskOrder.ts

```diff
@@ -0,0 +1,34 @@
+import { BaseEntity } from "./BaseEntity";
+import { Clin } from "./Clin";
+import { Column, Entity, ManyToOne, OneToMany } from "typeorm";
+import { FileScanStatus } from "../../models/FileMetadata";
+import { Portfolio } from "./Portfolio";
+
+@Entity()
+export class TaskOrder extends BaseEntity {
+  @Column({
+    type: String,
+    nullable: false,
+    length: 17,
+    comment: "TO numbers are 13 characters. TO modifications are 17 characters.",
+  })
+  taskOrderNumber: string;
+
+  @Column({ type: "uuid", nullable: false, comment: "S3 object key of task order pdf" })
+  fileId: string;
+
+  @Column({ type: String, nullable: false, length: 256, comment: "name of pdf file when uploaded" })
+  fileName: string;
+
+  @Column({ nullable: true, comment: "pdf file size in bytes" })
+  fileSize: number;
+
+  @Column({ type: "enum", nullable: false, enum: FileScanStatus, default: FileScanStatus.PENDING })
+  fileScanStatus: FileScanStatus;
+
+  @ManyToOne(() => Portfolio, (portfolio) => portfolio.taskOrders)
+  portfolio: Portfolio;
+
+  @OneToMany(() => Clin, (clin) => clin.taskOrder)
+  clins: Clin[];
+}
```

Comment:
Going through the examples here:
https://github.com/typeorm/typescript-example/blob/master/src/entity/Post.ts
I noticed that they used: 
```
@JoinTable()
```
Is that something we need?

Reply:
All of the relations that I created have the parent with a `@OneToMany` relationship with a child, and then in the child specifying the reverse `@ManyToOne` relationship.  This results in the child effectively having a foreign key column pointing to the parent.  So two entities, with the child pointing back to the parent.

A `@JoinTable` is not needed in these cases, as the example you've linked illustrates a `@ManyToMany` relationship.  Imagine that between the parent and child tables exists a third table, which is necessary to model that relationship in SQL.

> @JoinTable() is required for @ManyToMany relations. You must put @JoinTable on one (owning) side of relation.

Reply:
@zachclark-ccpo Good question.